onnx/shape_inference/implementation.cc

```diff
@@ -287,7 +287,7 @@ class ShapeInferenceImplBase {
       has_experimental_op = true;
     } else if (n.op_type() == "Constant" && n.output().size() == 1) {
       for (const auto& attr : n.attribute()) {
-        if (attr.name() == "value") {
+        if (attr.name() == "value" || attr.name() == "value_ints") {
```

Comment:
Thanks for creating the PR. Unfortunately, I don't think this will be enough to fix the issue. When the name is "value_ints", the attribute type is a list of ints, and not a tensor. So, the lines below also need to be extended to handle it correctly.

Reply:
Thanks for the suggestions! If i am correct, I need to add similar else if block which handles list of ints right?

Reply:
Yes. The proto fields are as defined here: https://github.com/onnx/onnx/blob/5b1346e30d66e4ec550f6b63c3883b258a2e8e3e/onnx/onnx.in.proto#L168
In addition, there is a need to convert the value into a TensorProto.

Reply:
Here is how onnxruntime does it: we need something similar: https://github.com/microsoft/onnxruntime/blob/4c37f15c1ba3e24e0c7e2110f34ab410fadb2340/onnxruntime/core/framework/tensorprotoutils.cc#L821